androidlibrary_lib/build.gradle

```diff
@@ -107,7 +108,8 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'androidx.fragment:fragment:1.3.0-beta01'
-
+    implementation 'com.google.android.material:material:1.4.0-alpha01'
```

Comment:
Please use the last stable version 1.3.0. I believe you are using things that work with the last stable release.